### ISSUE TITLE
Populate Found New Home

### ISSUE THREAD

**Issue by kcaponeAP** [activeprospect/leadconduit-integration-velocify#21]
In LCN we currently capture Mortgage Application Found Home. Please route this to 'Shopping Status' in Velocify. Attachment shows placement and allowed values.

Field name: found_new_home
![screen shot 2014-12-30 at 10 44 52 am](https://cloud.githubusercontent.com/assets/8238417/5580493/07bc5e9c-9011-11e4-83fa-2161417fef67.png)

**Comment by alexkwolfe** [comment]
What if we don't provide one of those options? Then what happens?

**Comment by kcaponeAP** [comment]
Then the field is not populated with any data at all.

**Comment by alexkwolfe** [comment]
Do we have a list of all the acceptable values for their fields? Or can we get that?

**Comment by kcaponeAP** [comment]
For every field? I will inquire. We've been tackling this on a case by case basis by logging into the system to inspect the existing field.

**Comment by alexkwolfe** [comment]
Yeah. I mean I'd think if they required specific values for specific fields then they'd be able to tell us what those are.

**Comment by kcaponeAP** [comment]
Direct from Velocify:
..  "We do not have a complete list of allowed values"

**Comment by cgrayson** [comment]
What's the status of this issue, is it still needed? 

If so, then: LC has `mortgage.application.is_home_found`, which is a true/false value. 

1. What Velocify field should that be sent as? (I see "Shopping Status", `found_new_home`, and `property_found` all mentioned above)
2.  What are the exact values they will accept in that field?

**Comment by cgrayson** [comment]
@kcaponeAP I should have said: those questions are for you.

**Comment by kcaponeAP** [comment]
1. Found new home and shopping status do not appear to be standard fields in Velocify. This is causing some confusion. In addition, the fields Shopping Status and Property Found contain more descriptive answers than Yes / No. 

2. Velocify won't supply us with a list of all values that they will accept in the fields. But here is the data from a customer's account. 

Property Found:
No, but I want to get Pre-Qualified or Pre-Approved
Yes
Yes, and I have signed a contract for this home
Yes, but I have not signed a contract for this home

Shopping Status:
Found a Home
Shopping

I think the fix for this is mapping until we see more usage of this.  I will take care of that on my end.
